coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -89,7 +89,9 @@ function AclApprovals() {
     mutationFn: approveAclRequest,
     onSuccess: (responses) => {
       const response = responses[0];
-      if (response.result !== "success") {
+      const responseIsAHiddenError =
```

Comment:
I wonder if we could extend `parseErrorMsg` to handle the case for the "200 errors", where the server returns a 200 but the status is not "success". It might simplify a lot of things, and we could provide helpful comments about this behaviour in a central place. Probably better left for another ticket though ^^

Reply:
Yessss, that's what I want to tackle today (or at least this week ^^ after our feature work). I want to sync with Backend first and understand the APIs better.